yelp_package/Makefile

```diff
@@ -24,12 +24,12 @@ DOCKER_RUN=docker run -t -v  $(CURDIR)/../:/work:rw paastatools_$*_container
 NOOP = true
 ifeq ($(PAASTA_ENV),YELP)
 	ADD_MISSING_DEPS_MAYBE:=-diff --unchanged-line-format= --old-line-format= --new-line-format='%L' ../requirements.txt ./extra_requirements_yelp.txt >> ../requirements.txt
-	ACTUAL_PACKAGE_VERSION:=$(RELEASE)-yelp1
-	ADD_YELP_PREFIX_MAYBE=dch -v $(ACTUAL_PACKAGE_VERSION) --force-distribution --distribution $* --changelog ../debian/changelog 'Build for yelp: add scribereader to virtualenv'
+	ACTUAL_PACKAGE_VERSION=$(RELEASE)-yelp1~$*1
```

Comment:
This will require updating our internal Puppet logic to allow pinning based on the Ubuntu release.